lib/new_relic/telemetry_sdk/clients/client.rb

```diff
@@ -9,7 +9,23 @@
 
 module NewRelic
   module TelemetrySdk
+    # This module contains most of the public API methods for the Ruby Telemetry SDK
```

Comment:
I don't think this line should be here. It implies that `Client` is like the `Agent` catch-all module in the Ruby agent, mainly a repository for API methods, but it's not.

Reply:
Pull it.  It was, more than anything, a test comment used to get yardoc up and running.